CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## v1.0.0 (2023-01-12)
+
+* deployments: updated mainnet deployment
+* contracts: major simplifications, fixes and optimizations
```

Comment:
I think it's worth to mention the merge of the resolution registry into the request manager explicitly

Reply:
Done.